src/main/java/uk/gov/ons/census/fwmt/csvservice/implementation/ccs/CCSCanonicalBuilder.java

```diff
@@ -1,38 +1,25 @@
 package uk.gov.ons.census.fwmt.csvservice.implementation.ccs;
 
-import uk.gov.ons.census.fwmt.canonical.v1.Address;
-import uk.gov.ons.census.fwmt.canonical.v1.CreateFieldWorkerJobRequest;
+import uk.gov.ons.census.fwmt.common.data.tm.SurveyType;
+import uk.gov.ons.census.fwmt.common.rm.dto.FwmtActionInstruction;
 import uk.gov.ons.census.fwmt.csvservice.dto.CCSPropertyListing;
 
 import java.util.UUID;
 
 public final class CCSCanonicalBuilder {
 
-  private static final String CREATE_ACTION_TYPE = "Create";
-
-  public static CreateFieldWorkerJobRequest createCCSJob(CCSPropertyListing ccsPropertyListing) {
-    Address address = new Address();
-    CreateFieldWorkerJobRequest createJobRequest = new CreateFieldWorkerJobRequest();
-
-    UUID createCaseId = UUID.randomUUID();
-
-    createJobRequest.setActionType(CREATE_ACTION_TYPE);
-    createJobRequest.setCaseId(createCaseId);
-    createJobRequest.setCaseReference(createCaseId.toString());
-    createJobRequest.setCaseType("CCSPL");
-    createJobRequest.setCoordinatorId(ccsPropertyListing.getCoordinatorId());
-    createJobRequest.setMandatoryResource(ccsPropertyListing.getCcsInterviewer());
-    createJobRequest.setSurveyType("CCS PL");
-    createJobRequest.setEstablishmentType("HH");
-    createJobRequest.setGatewayType(CREATE_ACTION_TYPE);
-    createJobRequest.setCategory("Not applicable");
-
-    address.setPostCode(ccsPropertyListing.getPostCode());
-    address.setLatitude(ccsPropertyListing.getLatitude());
-    address.setLongitude(ccsPropertyListing.getLongitude());
-    address.setOa(ccsPropertyListing.getOa());
-    createJobRequest.setAddress(address);
-
-    return createJobRequest;
+  public static FwmtActionInstruction createCCSJob(CCSPropertyListing ccsPropertyListing, int caseRefCount) {
+    return FwmtActionInstruction
+        .builder()
+        .caseId(String.valueOf(UUID.randomUUID()))
+        .caseRef("PL" + caseRefCount)
+        .surveyName("CCS_PL")
+        .addressType("CCS")
+        .surveyType(SurveyType.CCS_PL)
+        .fieldOfficerId(ccsPropertyListing.getFieldOfficerId())
+        .fieldCoordinatorId(ccsPropertyListing.getFieldCoordinatorId())
+        .postcode(ccsPropertyListing.getPostCode())
+        .oa(ccsPropertyListing.getOa())
+        .build();
```

Comment:
Does EstabType not need to be set 
https://collaborate2.ons.gov.uk/confluence/display/CATD/FWMT+Interfaces+-+File+Import+-+Census+Coverage+Survey

Reply:
think i use it for pciking up off qeueu